samples/03_org_administrators/validate_item_metadata.ipynb

```diff
@@ -0,0 +1,385 @@
+{
```

Comment:
<p>Thanks Manushi for adding this notebook!</p><p>Looking good to me except that I cannot view the output display of DataFrame or a list of sample <code>metadata</code> output.</p><p>Also, I believe the term <code>metadata</code> refers to this metadata property of the item dict. Can we either make some clarification that the metadata we are discussing here is not the same concept, or can we add one more column to the output data frame that also describes the <code>metadata</code> of a web item?    </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1566/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Thanks @CMPeng ! I updated the notebook to display first 5 rows of that dataframe and first 5 items of the `item_profile_status` dictionary. That way the user won't see everything happening in the geosaurus org. ;)

I also added a clarification for `metadata` in the first markdown cell, after the link to Enterprise Metadata documentation. Let me know if it needs any changes.